doc/contributing/primordials.md

```diff
@@ -387,8 +387,8 @@ SafePromiseAllReturnVoid([]); // safe
 SafePromiseAllReturnArrayLike([]); // safe
 
 const array = [promise];
-const set = new SafeSet().add(promise);
-// When running one of these functions on a non-empty iterable, it will also:
+const set = new SafeSet(array);
+// When running one of these functions on a non-empty non-array iterable, it will also:
```

Comment:
That's incorrect, `Promise.all` does rely on globally mutable `%Array.prototype%` no matter what iterable you pass to it, that's the point of section. Let's remove this change